chaoscenter/cypress/support/commands.js

```diff
@@ -0,0 +1,27 @@
+// Custom function for login
+Cypress.Commands.add('login', (Username,Password) => {
+    cy.visit('/login');
+    cy.get('input[name="username"]').type(Username);
+    cy.get('input[name="password"]').type(Password);
+    cy.intercept('POST','auth/login').as('login');
+    cy.get('.bp3-button').click({ });
```

Comment:
Can we check if there is some other selector we can use?
e.g. like `cy.get('button[label=""]')` or `cy.contains('button',"some text on button")`